sdk/cargo-build-sbf/src/main.rs

```diff
@@ -515,11 +515,12 @@ fn build_sbf_package(config: &Config, target_directory: &Path, package: &cargo_m
     env::set_var("OBJDUMP", llvm_bin.join("llvm-objdump"));
     env::set_var("OBJCOPY", llvm_bin.join("llvm-objcopy"));
 
+    let rustflags = env::var("RUSTFLAGS").ok();
+    let rustflags = rustflags.as_deref().unwrap_or("");
```

Comment:
nit: does this also work here?
```suggestion
    let rustflags = rustflags.as_deref().unwrap_or_default();
```

Reply:
it works. updated.